cache/cache.go

```diff
@@ -0,0 +1,104 @@
+package cache
+
+import (
+	"context"
+	"log"
+	"os"
+	"path"
+
+	"github.com/samsarahq/taskrunner"
+	"github.com/samsarahq/taskrunner/shell"
+)
+
+const CachePath = ".cache/taskrunner"
+
+var CacheDir = path.Join(os.Getenv("HOME"), CachePath)
```

Comment:
do we want this cached globally? Or do we want one per repo?

Reply:
I moved the logic for that out - it will be one per repo